pkg/kubectl/cmd/describe_test.go

```diff
@@ -168,3 +169,24 @@ func TestDescribeObjectSkipEvents(t *testing.T) {
 		t.Errorf("ShowEvents = false expected, got ShowEvents = %v", d.Settings.ShowEvents)
 	}
 }
+
+func TestDescribeHelpMessage(t *testing.T) {
+	f, _, _, _ := cmdtesting.NewAPIFactory()
+
+	buf := bytes.NewBuffer([]byte{})
+	buferr := bytes.NewBuffer([]byte{})
+	cmd := NewCmdDescribe(f, buf, buferr)
+	cmd.SetArgs([]string{"-h"})
+	cmd.SetOutput(buf)
+	_, err := cmd.ExecuteC()
+
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	expected := `describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)`
```

Comment:
why not `describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME) [options]` ?

Reply:
Good catch. Thanks.

Reply:
@shiywang `[options]` is added by kubectl template automatically, not belonging to `cobra`.

https://github.com/kubernetes/kubernetes/blob/c933067cd3cfa81349446b929816f732aa145b90/pkg/kubectl/cmd/templates/templater.go#L198-L205

This test will only cover testing `[flags]` in `useline`. No need to change to `describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME) [options]`.

Reply:
oh, sorry, my mistake, looks good

Reply:
wait, I still don't think this will catch the error, did a little test on my laptop
```
var t *testing.T
got := "describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME) [flags]"
expected := "describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)"
if !strings.Contains(got, expected) {
	t.Errorf("Expected to contain: \n %v\nGot:\n %v\n", expected, got)
}
```

Reply:
> maybe we only need to cover it doesn't contains [flags]

@shiywang make sense.